internal/common/compilation.bzl

```diff
@@ -292,6 +291,25 @@ def compile_ts(ctx,
   if not is_library:
     files += depset(tsickle_externs)
 
+  rerooted_es6_sources = []
+  es6_js_modules_roots = []
```

Comment:
You can remove the entire part about es6_js_module_roots and this still compiles. I think you should remove it for now because we obviously don't quite understand what this does correctly. As far as what I've read from Jart commenting on github, these roots are just stripped off from es6 import statements. I think they might only be applicable for absolute import statements (since Bazel creates a new file system)